dwh/bq_to_es_projector/dataset-summary_settings+mapping.json

```diff
@@ -0,0 +1,48 @@
+{
+  "settings": {
+    "analysis": {
+      "analyzer": {
+        "default": {
+          "type": "standard"
+        }
+      }
+    }
+  },
+  "mappings": {
+    "properties": {
+      "dataset_id": {
+        "type": "keyword"
+      },
+      "data_modalities": {
+        "type": "keyword"
+      },
+      "tissue_labels": {
+        "type": "keyword"
+      },
+      "cell_type_labels": {
+        "type": "keyword"
+      },
```

Comment:
We need to have some additional normalisers and analysers and it will also be beneficial to have additional "text" fields for fuzzy search